mod/thewire/views/default/thewire.js

```diff
@@ -7,7 +7,7 @@ elgg.thewire.init = function() {
 	var callback = function() {
 		var maxLength = $(this).data('max-length');
 		if (maxLength) {
-			elgg.thewire.textCounter(this, $("#thewire-characters-remaining span"), maxLength);
+			elgg.thewire.textCounter(this, $(this).closest('.elgg-form-thewire-add').find("#thewire-characters-remaining span").eq(0), maxLength);
```

Comment:
Are we confident the form was rendered by `elgg_view_form('thewire/add')`? Maybe just `form` without class selector?